specs/windows/interface_addresses.table

```diff
@@ -0,0 +1,11 @@
+table_name("interface_addresses")
```

Comment:
This seems simple enough to mimic `interface_addresses`: https://osquery.io/docs/tables/#interface_addresses

We can use the existing `interface` column for the Windows `interface_index`.